### ISSUE TITLE
Error in ogbn-proteins dataset

### ISSUE THREAD

**Issue by herman-nside** [danielegrattarola/spektral#90]
In line 35:
```python
graph, y = dataset[0]
```
the dict that is returned contains `None` for the key `node_feat`. 
This throws an error in line 36
```python
X, A, _ = ogb.graph_to_numpy(graph)
```
when it tries to determine
```python
graph['node_feat'].astype(dtype)
```

**Comment by danielegrattarola** [comment]
Hi, 

thanks for spotting this, all the examples for OGB were for a previous version. 
The example is now for the `ogbn-arxiv` dataset and replicates the example provided [here](https://github.com/snap-stanford/ogb/blob/master/examples/nodeproppred/arxiv/gnn.py).

Cheers, 
Daniele

**Comment by msaroufim** [comment]
There's still an issue with this example. I'll try and take a closer to see if anything changed with OGB

```
python ogbn-proteins_gcn.py 
2020-10-05 22:19:50.102665: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
Traceback (most recent call last):
  File "ogbn-proteins_gcn.py", line 37, in <module>
    X, A, _ = ogb.graph_to_numpy(graph)
  File "/home/ubuntu/GNN/env/lib/python3.7/site-packages/spektral/datasets/ogb.py", line 22, in graph_to_numpy
    E = graph['edge_feat'].astype(dtype)
AttributeError: 'NoneType' object has no attribute 'astype'
```

**Comment by danielegrattarola** [comment]
This should already be fixed in the latest version of Spektral.
Is this a PyPi installation or is it from source? I might need to update the PyPi version...

**Comment by danielegrattarola** [comment]
PyPi version was not up to date, it's fixed with Spektral 0.6.2. Can you give it a try?

**Comment by msaroufim** [comment]
It worked when I built from source, thanks @danielegrattarola ! 

I'll check the PyPi version later but I'm thinking I'll just use the source version from now on
